sqlmesh/core/macros.py

```diff
@@ -798,44 +806,48 @@ def safe_div(_: MacroEvaluator, numerator: exp.Expression, denominator: exp.Expr
         >>> from sqlmesh.core.macros import MacroEvaluator
         >>> sql = "SELECT @SAFE_DIV(a, b) FROM foo"
         >>> MacroEvaluator().transform(parse_one(sql)).sql()
-        'SELECT a / CASE WHEN b = 0 THEN NULL ELSE b END FROM foo'
+        'SELECT a / NULLIF(b, 0) FROM foo'
     """
-    return numerator / exp.Case().when(denominator.eq(0), exp.null()).else_(denominator)
+    return numerator / exp.func("NULLIF", denominator, 0)
```

Comment:
I thought this wasn't ANSI so we use case when?

Reply:
after further research, nullif seems ubiquitous